pages/wishlist.html

```diff
@@ -0,0 +1,162 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Wishlist-SudoMart</title>
+    <link rel="stylesheet" href="../css/wishlist.css">
+</head>
```

Comment:
Better to use your own component library for cards and other components

Reply:
Actually, I am importing my component lib in my common.css file and importing that common file in the entire project.